test/backend/inference_backend.py

```diff
@@ -1288,6 +1279,7 @@ def get_test_models():
         },
         # ==OP== GRU
         # ==MIN== 7
+        # ==LIM== Only `X` of inputs supports unknown dimensions.
```

Comment:
I know, in most cases, weights and biases are constants, so you are testing with X being unknown only. But it does not mean that this op does not support unknowns dimensions for weights and biases.  I though we wrote RNN ops using IndexExpr, and expected that it supports other unknown inputs also such as weights and biases. Could you please check it again? Thanks!

Reply:
I tested RNN, LSTM, and GRU with `DYNAMIC_SHAPE{{-1}: {-1}}`, but most of tests failed(10 out of 12 failed). I have not investigated them yet, but it seems we have issues in dynamic shape.

Reply:
Thanks for confirmation! You may not need to fix the issues but I would suggest to change the message to something like `W, B and R must be constants` to explicitly say the limitation applies to weights like W, B or R, not X.

Reply:
Done. Thanks!